index.js

```diff
@@ -275,9 +284,9 @@ export async function main () {
   // Path to push does not have to exist.
   // Non-existing path can be used for deletion.
   if (args.payload) {
-    for await (const result of push(args)) {
-      debugResult(result)
-    }
+    const result = push(args)
```

Comment:
The `push()` is an asynchronous generator. For the `result` to be returned, it needs to be awaited:
```js
const result = (await push(args).next()).value
```

Reply:
Yup forgot about that. Fixed